static/app/components/discover/performanceCardTable.tsx

```diff
@@ -1,100 +1,225 @@
         />
-      </UserMiseryPanelItem>
+      </TrendText>
     );
   }
 
-  sectionField(field: JSX.Element[]) {
+  function userMiseryTrend() {
+    const allReleasesUserMisery = allReleasesTableData?.data?.[0]?.user_misery;
+    const thisReleaseUserMisery = thisReleaseTableData?.data?.[0]?.user_misery;
     return (
       <StyledPanelItem>
-        <TitleSpace />
-        {field}
+        {renderChange(
+          allReleasesUserMisery as number,
+          thisReleaseUserMisery as number,
+          'number' as string
+        )}
       </StyledPanelItem>
     );
   }
 
-  renderFrontendPerformance() {
-    const vitals = [WebVital.FCP, WebVital.FID, WebVital.LCP, WebVital.CLS];
-    const webVitalTitles = vitals.map(vital => {
+  function renderFrontendPerformance() {
+    const webVitals = [
+      {title: WebVital.FCP, field: 'p75_measurements_fcp'},
+      {title: WebVital.FID, field: 'p75_measurements_fid'},
+      {title: WebVital.LCP, field: 'p75_measurements_lcp'},
+      {title: WebVital.CLS, field: 'p75_measurements_cls'},
+    ];
+
+    const spans = [
+      {title: 'HTTP', column: 'p75(spans.http)', field: 'p75_spans_http'},
+      {title: 'Browser', column: 'p75(spans.browser)', field: 'p75_spans_browser'},
+      {title: 'Resource', column: 'p75(spans.resource)', field: 'p75_spans_resource'},
+    ];
+
+    const webVitalTitles = webVitals.map((vital, idx) => {
       return (
-        <SubTitle key={vital}>
-          {WEB_VITAL_DETAILS[vital].name} ({WEB_VITAL_DETAILS[vital].acronym})
+        <SubTitle key={idx}>
+          <Link
+            to={{
+              pathname: discoverPath,
+              query: {
+                query: `${releaseEventView.query}`,
+                project: `${releaseEventView.project}`,
+                field: `p75(${vital.title})`,
+                statsPeriod: `${releaseEventView.statsPeriod}`,
+              },
+            }}
+          >
+            {WEB_VITAL_DETAILS[vital.title].name} (
+            {WEB_VITAL_DETAILS[vital.title].acronym})
+          </Link>
         </SubTitle>
       );
     });
 
-    const spans = ['HTTP', 'DB', 'Browser', 'Resource'];
-    const spanTitles = spans.map(span => {
-      return <SubTitle key={span}>{t(span)}</SubTitle>;
-    });
-
-    // TODO(kelly): placeholder data. will need to add discover data for webvitals and span operations in follow-up pr
-    const fieldData = ['0ms', '0ms', '0ms', '0ms'];
-    const field = fieldData.map(data => {
+    const spanTitles = spans.map((span, idx) => {
       return (
-        <Field key={data} align="right">
-          {data}
-        </Field>
+        <SubTitle key={idx}>
+          <Link
+            to={{
+              pathname: discoverPath,
+              query: {
+                query: `${releaseEventView.query}`,
+                project: `${releaseEventView.project}`,
+                field: `${span.column}`,
+                statsPeriod: `${releaseEventView.statsPeriod}`,
+              },
```

Comment:
You should be able to use `newView.getResultsViewUrlTarget(org.slug)` to do this. You will have to modify the event view first with `const newView = releaseEventView.withColumns()` to get a new view with just the column you're interested in showing.